src/set_theory/game/basic.lean

```diff
@@ -554,12 +571,69 @@ def inv' : pgame → pgame
   ⟨inv_ty l' r ff, inv_ty l' r tt,
     inv_val L' R IHl' IHr, inv_val L' R IHl' IHr⟩
 
-/-- The inverse of a surreal number in terms of the inverse on positive surreals. -/
+theorem inv'_pos : ∀ {x : pgame}, 0 < inv' x
+| ⟨xl, xr, xL, xR⟩ := by { convert lt_mk inv_ty.zero, refl }
+
+/-- `inv' 0` has exactly the same moves as `1`. -/
+def inv'_zero : relabelling (inv' 0) 1 :=
+begin
+  change relabelling (inv' (mk _ _ _ _)) 1,
+  rw inv',
+  refine ⟨_, _, λ i, _, is_empty_elim⟩,
+  { dsimp, apply equiv_punit_of_unique },
+  { dsimp, apply equiv.equiv_pempty },
+  { simp }
+end
+
+theorem inv'_zero_equiv : inv' 0 ≈ 1 := inv'_zero.equiv
+
+/-- `inv' 1` has exactly the same moves as `1`. -/
+def inv'_one : relabelling (inv' 1) 1 :=
+begin
+  change relabelling (inv' (mk _ _ _ _)) 1,
+  rw inv',
+  haveI H : is_empty {i : punit // (0 : pgame) < 0} := ⟨λ ⟨_, h⟩, pgame.lt_irrefl 0 h⟩,
+  refine ⟨_, _, λ i, _, is_empty_elim⟩,
+  { simp, apply @equiv_punit_of_unique _ (pgame.unique_inv_ty _ _),
+    { exact H },
+    { apply_instance } },
+  { dsimp, apply equiv.equiv_pempty },
+  { simp }
+end
+
+theorem inv'_one_equiv : inv' 1 ≈ 1 := inv'_one.equiv
+
+/-- The inverse of a pre-game in terms of the inverse on positive pre-games. -/
 noncomputable def inv (x : pgame) : pgame :=
-by classical; exact
-if x = 0 then 0 else if 0 < x then inv' x else inv' (-x)
+by { classical, exact if x ≈ 0 then 0 else if 0 < x then inv' x else inv' (-x) }
 
 noncomputable instance : has_inv pgame := ⟨inv⟩
 noncomputable instance : has_div pgame := ⟨λ x y, x * y⁻¹⟩
 
+theorem inv_equiv_zero {x : pgame} (h : x ≈ 0) : x⁻¹ = 0 :=
+by { apply if_pos, exact h }
+
+@[simp] theorem inv_zero : (0 : pgame)⁻¹ = 0 :=
+inv_equiv_zero (equiv_refl _)
+
+theorem inv_pos {x : pgame} (h : 0 < x) : x⁻¹ = inv' x :=
```

Comment:
```suggestion
theorem inv_eq_of_pos {x : pgame} (h : 0 < x) : x⁻¹ = inv' x :=
```
I'm trying to avoid `inv_of` becuase that's the name for `\frac1`, which is why I've added the `_eq`

Reply:
Should I also rename `inv_not_pos`?